CHANGELOG.md

```diff
@@ -1,3 +1,37 @@
+# 1.6.0-rc02 (February 2024)
+_Changes since 1.6.0-rc01_
+
+## Breaking changes
+_since 1.6.0-beta01_
+- When the resource library is used, resources should be imported explicitly. Replace this:
+```
+import <modulePackage>.generated.resources.Res
+```
+by this:
+```
+import <modulePackage>.generated.resources.Res
+import <modulePackage>.generated.resources.*
+```
+
+## iOS/desktop/web
+
+### Fixes
+- _(prerelease fix)_ [Fix `AlertDialog no work with ProvidableCompositionLocal`](https://github.com/JetBrains/compose-multiplatform-core/pull/1104)
```

Comment:
```suggestion
- _(prerelease fix)_ [Fix possible composition of `Dialog` before composition locals initialization](https://github.com/JetBrains/compose-multiplatform-core/pull/1104)
```

Reply:
It is implementation details. We should write from the user perspective